app/src/androidTest/java/org/systers/mentorship/LoginActivityTest.kt

```diff
@@ -0,0 +1,117 @@
+package org.systers.mentorship
+
+import android.support.annotation.IdRes
+import android.support.design.widget.TextInputLayout
+import android.support.test.espresso.Espresso.onView
+import android.support.test.espresso.ViewInteraction
+import android.support.test.espresso.action.ViewActions.*
+import android.support.test.espresso.assertion.ViewAssertions.matches
+import android.support.test.espresso.matcher.ViewMatchers.*
+import android.support.test.rule.ActivityTestRule
+import android.support.test.runner.AndroidJUnit4
+import android.view.View
+import android.widget.EditText
+import junit.framework.Assert.assertEquals
+import junit.framework.Assert.assertNotNull
+import kotlinx.android.synthetic.main.activity_login.*
+import org.hamcrest.Description
+import org.hamcrest.Matcher
+import org.hamcrest.Matchers.allOf
+import org.hamcrest.Matchers.not
+import org.hamcrest.TypeSafeMatcher
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.systers.mentorship.R.id.tiUsername
+import org.systers.mentorship.view.activities.LoginActivity
+
+/**
+ * This class specifies the UI test for LoginActivity
+ */
+@RunWith(AndroidJUnit4::class)
+class LoginActivityTest {
+
+    val EMPTY_USERNAME_ERROR : String = "Username cannot be empty"
+    val EMPTY_PASSWORD_ERROR : String = "Password cannot be empty"
+
+    @get:Rule
+      var mActivityTestRule : ActivityTestRule<LoginActivity> = ActivityTestRule(LoginActivity::class.java)
+
```

Comment:
Try to remove excessive blank lines in this document.

Reply:
@isabelcosta I have made all the changes requested by you :+1:  Have a look

Reply:
Cool